ballerina/xmldata.bal

```diff
@@ -41,7 +41,7 @@ public type JsonOptions record {
 # + jsonValue - The JSON source to be converted to XML
 # + options - The `xmldata:JsonOptions` record for JSON to XML conversion properties
 # + return - XML representation of the given JSON if the JSON is
-# successfully converted or else an `xmldata:Error`
+# successfully converted or else an `xmldata:Error`. The `()` value is not returned
```

Comment:
Why include `()` in the return type if it's never returned by the function though?